sampler_test.go

```diff
@@ -553,11 +542,8 @@ func TestRemotelyControlledSampler_updateSamplerFromAdaptiveSampler(t *testing.T
 	// Overwrite the sampler with an adaptive sampler
```

Comment:
^^ `defer adaptiveSampler.Close()` - weird. Note that we don't close the samplers when re-assigning them in the Remote Sampler. Ideally only remote sampler would have the Close method to begin with, i.e. we could remove it from Sampler interface and just do a cast-check to io.Closer in the tracer. <end of rambling> Not in this diff